src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise.component.ts

```diff
@@ -103,6 +108,25 @@ export class FileUploadExerciseComponent extends ExerciseComponent {
         });
     }
 
+    /**
+     * Deletes all the given file upload exercises
+     * @param exercisesToDelete the exercise objects which are to be deleted
+     * @param event contains additional checks which are performed for all these exercises
+     */
+    deleteMultipleFileUploadExercises(exercisesToDelete: FileUploadExercise[]) {
```

Comment:
I also thought about this. But since I adapted the standard `delete` method and it is also very similar in these files, I decided to stick to this "pattern". I thought this would be a possible refactoring which is out of scope for this PR. What do you think?

Reply:
I'd appreciate if you used inheritance at least for the newly added methods